flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -1121,13 +1117,14 @@ private void testFailToConfirmCheckpointMessage(Consumer<StreamTask<?, ?>> consu
         StreamTaskMailboxTestHarness<Integer> harness =
                 builder.setupOutputForSingletonOperatorChain(streamMap).build();
 
-        try {
-            consumer.accept(harness.streamTask);
-            harness.streamTask.runMailboxLoop();
-            fail();
-        } catch (ExpectedTestException expected) {
-            // expected exceptionestProcessWithUnAvailableInput
-        }
+        // expected exceptionestProcessWithUnAvailableInput
+        assertThatThrownBy(
+                        () -> {
+                            consumer.accept(harness.streamTask);
+                            harness.streamTask.runMailboxLoop();
+                            fail(null);
```

Comment:
Why we need `fail`?

Reply:
It's from the old line.
And I removed it.